develop/effects.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Effects
+description: Adding a custom effect.
+authors:
+  - dicedpixels
+  - YanisBft
+  - FireBlast
+  - Friendly-Banana
+  - SattesKrokodil
+---
+
+<!-- Couldn't find GitHub usernames for: siglong, tao0lu  -->
+
+# Effects
+
+Effects are a condition that can affect an entity. They can be positive, negative or neutral in nature. The base game
+applies these effects in various ways such as food, potions etc.
+
+The `/effect` command can be used to apply effects on an entity.
+
+## Custom Effects
+
+In this tutorial we'll add a new custom effect called _Tater_ which gives you one experience point every game tick.
+
+### Extend `StatusEffect`
+
+Let's create a custom effect class by extending `StatusEffect`, which is the base class for all effects.
+
+@[code lang=java transcludeWith=:::1](@/reference/latest/src/main/java/com/example/docs/effect/TaterEffect.java)
+
+### Registering Your Custom Effect
+
+Similar to block and item registration, we use `Registry.register` to register our custom effect into the
+`STATUS_EFFECT` registry. This can be done in our initializer.
+
+@[code lang=java transcludeWith=:::1](@/reference/latest/src/main/java/com/example/docs/effect/FabricDocsReferenceEffects.java)
```

Comment:
Might want to add a callout that links to the "Potions" section, eg: "If you want to add a custom potion item for your effect, checkout the [Potions] page."